Reference/places_mapdata.json

```diff
@@ -0,0 +1,1472 @@
+{
+  "featureId": "labels-abandoned-mines",
+  "categoryId": "wynntils:place:town-or-place",
```

Comment:
I think we should just rename this to `wynntils:place:place`. I know "town or place" is more descriptive, since we really mix "places" and "towns" for this category, but this long name is also a bit silly.

... or ... *thinking* ... maybe we should split it up in `town` and `place`. It will require manual classification of towns (Bucie, etc), but it would really be much better.

Reply:
place:place is a bit, weird? Not sure how to call it. I would be able to live with town-or-place, but I don't really want to have manual separation of the data.